lib/galaxy/web/stack/__init__.py

```diff
@@ -337,10 +338,12 @@ def log_startup(self):
         msg = ["Galaxy server instance '%s' is running" % self.app.config.server_name]
         # the last worker isn't guaranteed to start last, but it's the best shot
         if not self._is_mule and self.instance_id == len(self.workers()):
+            # We use the same text printed by Paste to not break old scripts grepping for this line
+            msg.append('Starting server in PID %d.' % os.getpid())
```

Comment:
I am not sure if it's a big deal but this pid is only going to be for the last worker. We could have all forked processes + the master log their pids, but uWSGI already does this, just in another format.

Reply:
Which one gets written to `galaxy.pid` if you start Galaxy with `./run.sh --pidfile galaxy.pid` ?

Reply:
To answer myself, it's the pid of the master process (if using `master: true`) or the one of the first worker.